Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -942,6 +942,17 @@ private async void OnPageChanged( int currentPage )
 
     #region Commands
 
+    /// <summary>
+    /// Sets the DataGrid into the loading state. 
+    /// <para>Makes sure to invoke the StateHasChanged method.</para>
+    /// </summary>
+    /// <param name="isLoading">Whether the grid is loading or not</param>
+    public void SetLoading(bool isLoading)
+    {
+        IsLoading = isLoading;

```

Comment:
Should be also make IsLoading public? But only the `get` part.

Reply:
Done.